### DISCUSSION TITLE
Training for Greek + English language

### DISCUSSION THREAD

**Discussion by Ham714** [PaddlePaddle/PaddleOCR#8908] [Q&A]
I am trying to train recognition model for Greek +English language. Well the training is continue, its been 2 days now the accuracy is reaching to 67% and when i  export the model and test it on some other image it giving empty box.
i am using CPU based hardware 

this is my train.log
[train.log](https://github.com/PaddlePaddle/PaddleOCR/files/10516611/train.log)
this is my configuration file
[config.txt](https://github.com/PaddlePaddle/PaddleOCR/files/10516625/config.txt)

I have generated 10000 images for training and 1000 images for validation
if someone want dataset i can post that here too

the python script I am using is this and the result is empty box
![image](https://user-images.githubusercontent.com/119650336/215034368-471ab01e-c5f7-4140-be85-0dc2dd6c10a1.png)
![image](https://user-images.githubusercontent.com/119650336/215034388-9f11b3d1-3980-4708-bf1a-b5b916b03807.png)
 the image on which i am testing is this 

![image3_2ndp](https://user-images.githubusercontent.com/119650336/215034476-fda6c5a9-f3b5-4f45-bcaa-409289f2d16e.jpg)
 
can anyone guide me on this plz @Evezerest @Topdu

**Comment by Topdu** [comment]
Trying to add paramters plz: rec_algorithm='SVTR', rec_image_shape='3,64,256'.
